app/helpers/application_helper.rb

```diff
@@ -1,2 +1,15 @@
 module ApplicationHelper
+  #for nice notifications with toastr
+  def custom_bootstrap_flash
+    flash_messages = []
+    flash.each do |type, message|
+      type = 'success' if type == 'notice'
+      type = 'error'   if type == 'alert'
+      puts type
+
+      text = "<script>toastr.#{type}('#{message}');</script>"
```

Comment:
Please escape the message for now, (The helper is called `j`) if we run into ugly messages I would rather we think of something to pass through some html than having a possible XSS vector.